content/help/security-vulnerabilities/index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Istio Security Vulnerability Disclosure Information
+description: Responsible disclosure for Istio security vulnerabilities
+weight: 35
+aliases:
+    - /security-vulnerabilities.html
+    - /security-vulnerabilities/index.html
+---
+## How to report a security vulnerability?
+
+We’re very grateful for security researchers and users that report Istio security vulnerabilities to us. We will thoroughly investigated every report. 
```

Comment:
Line length and grammar. I suggest:
We are very grateful to the security researchers and users that report back Istio security vulnerabilities. We investigate every report thoroughly.

Reply:
Done.